platform/api-server/api/routes.js

```diff
@@ -26,13 +26,17 @@ module.exports = function(app) {
     app.get('/gateway/apps', gatewayAPIController.getApps);
     app.get('/gateway/link-graph-data', gatewayAPIController.getLinkGraphData);
     app.post('/gateway/execute-app', uploader.fields([{name: 'app'}, {name: 'metadata'}]),
-        gatewayAPIController.executeApp);
+             gatewayAPIController.executeApp);
+    app.post('/gateway/execute-app-v2',
```

Comment:
i think you can remove the previous version and call this `platform/execute-app`. if there's some specific reason why you have maintained both we can discuss.